src/praisonai-agents/praisonaiagents/memory/enhanced_memory.py

```
@@ -0,0 +1,647 @@
+                conn.close()
+                
+                results = []
+                for row in rows:
+                    results.append({
+                        "id": row[0],
+                        "content": row[1],
+                        "metadata": json.loads(row[2] or "{}"),
+                        "created_at": row[3]
+                    })
+                return results
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)